Assets/Scripts/Character.cs

```diff
@@ -39,6 +39,8 @@ enum State
     Vector2 startPos;
     Vector2 endPos;
 
+    public CapsuleCollider playerCollider;
+
```

Comment:
Make this private Serialized field and make a getter

Reply:
Done